### PR TITLE
Add plugin-cachet-gating-plugin.yml (https://github.com/jenkinsci/cachet-gating-plugin)

### PR DESCRIPTION
**Pull request by ggallen**
# Description

https://github.com/jenkinsci/cachet-gating-plugin

# Submitter checklist for changing permissions

### Always

- [X] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [X] Make sure the file is created in `permissions/` directory
- [X] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [X] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [\X] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [X] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [X] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by daniel-beck** [comment]
Please complete all items in the checklist and check those that you did.

This is a newly hosted plugin, so please also provide the reference to the HOSTING issue.

**Comment by olivergondza** [comment]
@daniel-beck, the repo and JIRA component ware created over IRC bot so there is no hosting request.

@ggallen, please `s/ogondza/olivergondza/` as that is my LDAP uid.

**Comment by daniel-beck** [comment]
@olivergondza Right, which explains why it does not follow https://jenkins.io/doc/developer/publishing/style-guides/#artifact-id

**Comment by ggallen** [comment]
@daniel-beck , is there anything else I need to do here? Thanks!

**Comment by batmat** [comment]
@ggallen PR build is failing, did you have a look :)? There's still a typo on oliver Jenkins id... 
(note: please, that's great if you review failing PR builds before pinging people around :))

Thanks!

**Comment by ggallen** [comment]
@daniel-beck , I did have a look. From what I see the archive was failing and the error there made no sense to me.  The run step supposedly passed, but I see from looking at the output that it did in fact fail.

I will fix the typo.  And next time I'll check all the steps regardless of the status!

**Comment by daniel-beck** [comment]
No LICENSE file that would allow GitHub to determine license status 😞

File: permissions/plugin-cachet-gating.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+name: "cachet-gating"
+github: "jenkinsci/cachet-gating-plugin"
+paths:
+- "org/jenkins-ci/plugins/cachet-gating-plugin"
```
Comment (daniel-beck) [inline permissions/plugin-cachet-gating.yml]: This is still wrong.

**Pull request merged**